analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextBaseTest.ShouldAnalyzeTree.cs

```diff
@@ -150,12 +150,12 @@ public void ShouldAnalyzeTree_CorrectSettingUsed(string fileName, bool expectedC
         sutCS.ShouldAnalyzeTree(treeCS, CSharpGeneratedCodeRecognizer.Instance).Should().Be(expectedCSharp);
         sutVB.ShouldAnalyzeTree(treeVB, VisualBasicGeneratedCodeRecognizer.Instance).Should().BeTrue();
 
-        sonarLintXml.ToStringCallCount.Should().Be(2, "file should be read once per language");
+        sonarLintXml.ToStringCallCount.Should().Be(3, "file should be read once per language"); // TEMPORARILY BUMPED TO 3
 
         // Read again to check caching
         sutVB.ShouldAnalyzeTree(treeVB, VisualBasicGeneratedCodeRecognizer.Instance).Should().BeTrue();
 
-        sonarLintXml.ToStringCallCount.Should().Be(2, "file should not have been read again");
+        sonarLintXml.ToStringCallCount.Should().Be(3, "file should not have been read again"); // TEMPORARILY BUMPED TO 3
```

Comment:
This bump will be reverted once the new SonarLintXmlReader will be used to access all properties.